### ISSUE TITLE
`Message: no such element: Unable to locate element: {"method":"xpath","selector":"/html/body/div[2]/div/div[2]/div[1]/div[1]/h1"}`

### ISSUE THREAD

**Issue by nilsreichardt** [Valentin-Metz/tum_video_scraper#8]
```
Nils-MBP-13-M1-3:tum_video_scraper nils$ python3 src/main.py -c config.yaml 
Starting new run!

Scanning TUM-live:
Traceback (most recent call last):
  File "/Users/nils/Desktop/Projects/tum_video_scraper/src/main.py", line 150, in <module>
    tum_live.get_subjects(tum_live_subjects, username, password, videos_for_subject)
  File "/Users/nils/Desktop/Projects/tum_video_scraper/src/tum_live.py", line 74, in get_subjects
    m3u8_playlists = get_video_links_of_subject(driver, subjects_identifier, camera_type)
  File "/Users/nils/Desktop/Projects/tum_video_scraper/src/tum_live.py", line 49, in get_video_links_of_subject
    filename = driver.find_element(By.XPATH, "/html/body/div[2]/div/div[2]/div[1]/div[1]/h1").text.strip()
  File "/Library/Frameworks/Python.framework/Versions/3.9/lib/python3.9/site-packages/selenium/webdriver/remote/webdriver.py", line 857, in find_element
    return self.execute(Command.FIND_ELEMENT, {
  File "/Library/Frameworks/Python.framework/Versions/3.9/lib/python3.9/site-packages/selenium/webdriver/remote/webdriver.py", line 435, in execute
    self.error_handler.check_response(response)
  File "/Library/Frameworks/Python.framework/Versions/3.9/lib/python3.9/site-packages/selenium/webdriver/remote/errorhandler.py", line 247, in check_response
    raise exception_class(message, screen, stacktrace)
selenium.common.exceptions.NoSuchElementException: Message: no such element: Unable to locate element: {"method":"xpath","selector":"/html/body/div[2]/div/div[2]/div[1]/div[1]/h1"}
  (Session info: headless chrome=108.0.5359.124)
Stacktrace:
0   chromedriver                        0x0000000102b77b7c chromedriver + 4209532
1   chromedriver                        0x0000000102b03418 chromedriver + 3732504
2   chromedriver                        0x00000001027b6368 chromedriver + 271208
3   chromedriver                        0x00000001027f1fc8 chromedriver + 516040
4   chromedriver                        0x0000000102829804 chromedriver + 743428
5   chromedriver                        0x00000001027e459c chromedriver + 460188
6   chromedriver                        0x00000001027e5644 chromedriver + 464452
7   chromedriver                        0x0000000102b4930c chromedriver + 4018956
8   chromedriver                        0x0000000102b4cfd8 chromedriver + 4034520
9   chromedriver                        0x0000000102b5311c chromedriver + 4059420
10  chromedriver                        0x0000000102b4dbac chromedriver + 4037548
11  chromedriver                        0x0000000102b27bb0 chromedriver + 3881904
12  chromedriver                        0x0000000102b6a120 chromedriver + 4153632
13  chromedriver                        0x0000000102b6a274 chromedriver + 4153972
14  chromedriver                        0x0000000102b7e82c chromedriver + 4237356
15  libsystem_pthread.dylib             0x00000001a8d9c26c _pthread_start + 148
16  libsystem_pthread.dylib             0x00000001a8d9708c thread_start + 8
```

My `config.yaml`

```yaml
TUM-live:
  "EidI": "2022/W/eidi:COMB"

Username: "..."
Password: "..."

Output-Folder: "...."

Maximum-Parallel-Downloads: 3
```

**Comment by Valentin-Metz** [comment]
Fixed in 31cc188b23a41426f101ee7e458474126e4f509a.

**Comment by nilsreichardt** [comment]
Cool, thanks for the fix! ❤️
